tests/images/Dockerfile.canary.china

```diff
@@ -0,0 +1,35 @@
+FROM ubuntu:18.04
```

Comment:
I understand this is a simplified version of the other container, but I don't think we should be duplicating code. It would be preferable to extend the other image (or simply use the other image) if all we are doing is changing the entrypoint? 

This can be done by either:
1. Building the China image with the US image as the base
2. Running the US image with a different entrypoint when doing `docker run`.

Reply:
Yep 
once we have same infra in US and China region we can just use 1 Dockerfile for both 

-------
Currently there are some issues like
 * sagemaker-k8s-operator.tar.gz doesn't exist for China canary 
 * Kustomize fails to download